.github/workflows/docker.yml

```diff
@@ -1,6 +1,6 @@
 name: Push docker images
 env:
-  BASE_IMAGE: "ubuntu:18.04"
```

Comment:
I bumped this to `20.04` in #5432 as an intermediate step until this lands.

I wish there was a way to programmatically pull this from the `Dockerfile`, but not coming up with any easy solutions... everything I think of seems very brittle/hacky. Anyone else got an idea?